handlers/handler-fault-injection/src/main/java/org/apache/servicecomb/faultinjection/Fault.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.faultinjection;
+
+import org.apache.servicecomb.core.Invocation;
+
+public interface Fault {
+
+  int getPriority();
+
+  FaultResponse injectFault(Invocation invocation, FaultParam faultAttributes);
```

Comment:
I think this interface is not very good, it'll be changed when new Faults is added. 
1. FaultParam is implementation dependent, we do not know which param need to pass to specific fault implementation
2. FaultResponse is better to be an AsynCallback in order to handle deadlock problems mentioned in other review comments.